Dockerfile.prod

```diff
@@ -7,6 +7,8 @@ WORKDIR /app
 COPY .env .eslintrc.js package.json yarn.lock rollup.config.js /app/
 RUN yarn
 
+ENV HUSKY_SKIP_INSTALL=1
```

Comment:
This should be before the `yarn` command to take effect, as it only applies to the subsequent instrucions